src/bullsAndCowsFunction.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+function bullsAndCows(userNumberMeaning, randomNumberMeaning) {
+  let bulls = 0;
+  let cows = 0;
+
+  for (let i = 0; i < 4; i++) {
```

Comment:
4 is a magic number, so make sense to move it to the constant 
check other places